sys/net/gnrc/application_layer/tftp/gnrc_tftp.c

```diff
@@ -606,6 +608,7 @@ tftp_state _tftp_state_processes(tftp_context_t *ctxt, msg_t *m)
     tmp = gnrc_pktsnip_search_type(pkt, GNRC_NETTYPE_IPV6);
     ipv6_hdr_t *ip = (ipv6_hdr_t *)tmp->data;
     uint8_t *data = (uint8_t *)pkt->data;
+    ctxt->dst_port = byteorder_ntohs(udp->src_port);
```

Comment:
I'm not 100% sure about this one. There are also other sets of this variable below... However, I observed `gnrc_udp_hdr_build()` getting `dst_port` 0 with the fix above.

Reply:
I pushed a fixup that I _think_ is more correct.